src/utils/send-request.js

```diff
@@ -44,7 +56,7 @@ function onRes (buffer, cb) {
     }
 
     if (res.statusCode >= 400 || !res.statusCode) {
-      return parseError(res, cb)
+      return parseError(res, cb, isJson)
```

Comment:
See previous comment for explanation :)

```suggestion
      return parseError(res, isJson, cb)
```